dpe/Cargo.toml

```diff
@@ -20,3 +21,4 @@ asn1 = "0.13.0"
 openssl = "0.10"
 x509-parser = "0.14.0"
 crypto = {path = "../crypto", features = ["deterministic_rand"]}
+x509 = {path = "../x509", features = ["dpe_profile_p256_sha256"]}
```

Comment:
I would recommend reversing this dependency. Instead, above you can do something like

```
[features]
default = ["dpe_profile_p256_sha256"]
dpe_profile_p256_sha256 = ["x509/dpe_profile_p256_sha256"]
dpe_profile_p384_sha384 = ["x509/dpe_profile_p384_sha384"]
```